common/containers/Tabs/ENS/components/generalInfoPanel/helperStatelessFunctionals.jsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import type { ABlankNooperProps } from './types';
+
+export const ABlankNooper = ({
+  content, //eslint-disable-line
+  href //eslint-disable-line
```

Comment:
What's eslint beefing about here?

Reply:
Throwing an error that I dont have prop-type validation on it, it's related to #122. Once it's finished I won't need those disabled lines